go/api/base_client.go

```diff
@@ -3062,3 +3062,168 @@ func (client *baseClient) XAck(key string, group string, ids []string) (int64, e
 	}
 	return handleIntResponse(result)
 }
+
+// Changes the ownership of a pending message.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//	key      - The key of the stream.
+//	group    - The name of the consumer group.
+//	consumer - The name of the consumer.
+//	minIdleTime - The minimum idle time in milliseconds.
+//	ids        - The ids of the entries to claim.
+//
+// Return value:
+//
+//	A `map of message entries with the format `{"entryId": [["entry", "data"], ...], ...}` that were claimed by
+//	the consumer.
+//
+// Example:
+//
+// [valkey.io]: https://valkey.io/commands/xclaim/
+func (client *baseClient) XClaim(
+	key string,
+	group string,
+	consumer string,
+	minIdleTime int64,
+	ids []string,
+) (map[string][][]string, error) {
+	result, err := client.executeCommand(
```

Comment:
Rather than duplicating code, just pass a `nil` options value to `XClaimWithOptions` and handle the `nil` to create the appropriate `executeCommand` call.